snippets/quantity-input.liquid

```diff
@@ -33,6 +33,15 @@
     id="Quantity-{{ variant.id }}"
     data-index="{{ variant.id }}"
   >
+  <noscript>
```

Comment:
The reason I needed a second input instead of the existing one was that I had to remove parameter `min` which didn't allow us to input 0 qty. When we update the variant list some of the variants may remain with 0 qty and for that reason we cannot use parameter `min` anymore. 

However, I forgot that I added parameter `formnovalidate` to the submit button and now I can remove this second input because I'm not validating form anymore. Good call 👍

Reply:
The `min` exists in cart and we dont duplicate it there 👀 But yes, let me know once it's removed :)

Reply:
For whoever will read this PR I leave a video with some explanations here :)
https://screenshot.click/14-53-5zr6l-vvpoe.mp4